src/main/java/ninjabrainbot/gui/panels/main/ThrowPanel.java

```diff
@@ -114,7 +114,7 @@ void setAngleErrorsEnabled(boolean e) {
 	}
 
 	private void updateError(ChunkPrediction p) {
-		error.setText(t == null || p == null ? null : String.format(Locale.US, "%.3f", p.getAngleError(t)));
+		error.setText(t == null || p == null ? null : String.format(Locale.US, "%.4f", p.getAngleError(t)));
```

Comment:
Dont change existing behaviours for users who dont use this feature, showing this amount of information is redundant to them. You should only display 4 significant figures here if the use tall res setting is enabled.